README.md

```diff
@@ -42,7 +43,7 @@ You can get either a binary distribution of *eprosima Fast RTPS* or compile the
 The latest, up to date binary release of *eprosima Fast RTPS* can be obtained from the <a href='http://www.eprosima.com'>company website</a>.
 
 ### Installation from Source
-To compile *eprosima Fast RTPS* from source, at least Cmake version 2.8.12 is needed.
+To compile *eprosima Fast RTPS* from source, at least Cmake version 2.8.12 and Boost 1.61 are needed.
```

Comment:
Did somebody add boost dependency?

Reply:
No, I think I just got confused with the README.md of Fast-RTPS-docs. Removed.